### DISCUSSION TITLE
Bro, Me and you have same processor (mine 5600g), same graphic and exact same Motherboard. Add instruction for setup

### DISCUSSION THREAD

**Discussion by SonaliBendre** [sileshn/Ryzentosh#29] [General]
In Readme add instruction how to install in Gigabyte mobo.
Also please tell what to change in mobo

**Comment by sileshn** [comment]
We don't have the same specs. You have an APU. If you are using an additional graphics card, you will have to disable the iGPU of 5600g or use [NootedRed](https://github.com/NootInc/NootedRed) without the dGPU.

I do not have any experience of configuring on APU's, so you will get better support on the AMD discord server.
